components/ModInputField.vue

```diff
@@ -41,4 +39,10 @@ const checkValidationOfInputField =(inputValue:string) => {
     isTheInputValueValid.value = props.inputValidationCheck(inputValue)
 }
 
+const handleInput = (event: Event) => {
```

Comment:
not really related to this PR context, but I'm fine with improvements 😉

Reply:
I guess unless the compiler is complaining

Reply:
The typescript compiler was complaining about the type so this was the best practice I could find to fix it